packages/connector-sendgrid-mail/README.md

```diff
@@ -1,2 +1,86 @@
-### SendGrid Mail README
-placeholder
+# SendGrid mail connector
+
+SendGrid (also known as Twilio SendGrid) is a customer communication platform for transactional and marketing email. We can use its email sending function to send a "verification code" (or in other words, passcode).
+
+## Register and sign in SendGrid account
+
+Go to https://app.sendgrid.com/ to register a new account and sign in to SendGrid afterward. You may skip the registration step if you've already got an account.
```

Comment:
add a link?

Reply:
Simply: `Sign in or create a new account at XXXXX. `